gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/EcsClusterController.groovy

```diff
@@ -0,0 +1,38 @@
+/*
+  * Copyright 2014 Netflix, Inc.
+  *
+  * Licensed under the Apache License, Version 2.0 (the "License");
+  * you may not use this file except in compliance with the License.
+  * You may obtain a copy of the License at
+  *
+  *    http://www.apache.org/licenses/LICENSE-2.0
+  *
+  * Unless required by applicable law or agreed to in writing, software
+  * distributed under the License is distributed on an "AS IS" BASIS,
+  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  * See the License for the specific language governing permissions and
+  * limitations under the License.
+  */
+
+package com.netflix.spinnaker.gate.controllers
+
+import com.netflix.spinnaker.gate.services.EcsClusterService
+import io.swagger.annotations.ApiOperation
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RequestMethod
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+@RequestMapping("/ecsClusters")
+class EcsClusterController {
+  @Autowired
+  EcsClusterService ecsClusterService
+
+  @ApiOperation(value = "Retrieve a list of ECS clusters that can be used by the selected cloud provider")
+  @RequestMapping(value = "/{provider}/{account}/{region}", method = RequestMethod.GET)
```

Comment:
I don't think other cloud providers can use these ECS clusters

Reply:
Fixed.
The path no longer needs `/{provider}/`.